config_scheduler.go

```diff
@@ -118,6 +120,14 @@ func (c *SchedulerConfig) ValidateAndDefault() error {
 		return errors.Errorf("supported host allocator feedback rules are %s; %s is not supported",
 			ValidHostAllocatorFeedbackRules, c.HostAllocatorFeedbackRule)
 	}
+	if c.HostsOverallocatedRule == "" {
+		c.HostsOverallocatedRule = HostsOverallocatedIgnore
+	}
+
+	if !utility.StringSliceContains(ValidDefaultHostsOverallocatedRules, c.HostsOverallocatedRule) {
```

Comment:
I know the other rules also do this but it seems odd that we check if the rule is in listA but then in the error we return listB (which contains "", while listA does not)

Reply:
I fixed the others too